packages/react-core/src/demos/DataList/examples/DataListPagination.tsx

```diff
@@ -0,0 +1,315 @@
+import React from 'react';
+import {
+  Button,
+  DataList,
+  DataListItem,
+  DataListCell,
+  DataListItemRow,
+  DataListItemCells,
+  Flex,
+  FlexItem,
+  MenuToggle,
+  MenuToggleCheckbox,
+  OverflowMenu,
+  OverflowMenuControl,
+  OverflowMenuItem,
+  PageSection,
+  PageSectionVariants,
+  Pagination,
+  Text,
+  TextContent,
+  TextVariants,
+  Toolbar,
+  ToolbarItem,
+  ToolbarContent
+} from '@patternfly/react-core';
+import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon';
+import { DashboardWrapper } from '@patternfly/react-core/dist/js/demos/DashboardWrapper';
+
+import CodeBranchIcon from '@patternfly/react-icons/dist/esm/icons/code-branch-icon';
+import CodeIcon from '@patternfly/react-icons/dist/esm/icons/code-icon';
+import CubeIcon from '@patternfly/react-icons/dist/esm/icons/cube-icon';
+import ExclamationTriangleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-triangle-icon';
+import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
+import TimesCircleIcon from '@patternfly/react-icons/dist/esm/icons/times-circle-icon';
+
+export const DataListPagination: React.FunctionComponent = () => {
+  const renderPagination = () => <Pagination itemCount={5} page={1} variant="top" isCompact />;
```

Comment:
Pagination auto-generates an `id` so the a11y issue is because the same id is getting made from the two calls. Passing a different `id` prop to the top/bottom paginations should fix it.

Reply:
We should do something similar to the [table bulk select demo](https://patternfly-react-pr-10276.surge.sh/components/table/react-demos#bulk-select), where the function that returns a Pagination accepts args for the variant and isCompact value. I imagine the first pagination should follow other demos, where it `variant="top" isCompact={false}`, but the pagination at the bottom of the page should have something more like `variant="bottom" isCompact={true} isSticky={variant === 'bottom'}`.

Passing the different variants in wil also resolve the id issue.

We would probably want to also add functionality to change pages and select the amount of items to show per page, but that's partially dependent on if we add enough items to trigger pagination